org.eclipse.xtext.ide.tests/testlang-src/org/eclipse/xtext/ide/tests/testlanguage/rename/TestLanguageRenameService.java

```diff
@@ -56,4 +62,11 @@ protected EObject getElementWithIdentifierAt(XtextResource xtextResource, int of
 		}
 		return null;
 	}
+
+	protected boolean isIdentifier(ILeafNode leafNode) {
```

Comment:
This is no longer used, is it?

Reply:
It was still used but I have removed the usage now, as we can reuse the implementation of `EObjectAtOffsetHelper#getElementWithNameAt`